### DISCUSSION TITLE
Siege addon feature ideas

### DISCUSSION THREAD

**Discussion by SilverGXjs** [GriefPrevention/GriefPrevention#15] [Ideas]
Hey i'm searching for an option in the Config.yml.... is it possible to set  the Siege Cooldown Timer ?
Or maybe is it possible to add this feature / option.
And i miss an Option to set that you can take thinks out of a chest but nothin in...so you can prepare for battle when you got be sieged....

**Comment by SilverGXjs** [comment]
And sry for spamming... did i miss the command for Kicking players out of yout Claim... so when other players can jump infront of you so you can Kick and Lock them out of your claim...

**Comment by RoboMWM** [comment]
> did i miss the command for Kicking players out of yout Claim... so when other players can jump infront of you so you can Kick and Lock them out of your claim...

https://www.spigotmc.org/resources/griefprevention-flags-from-the-creator-of-griefprevention.10280/

As for siege, I'd rather this feature be an addon vs. in the plugin itself. I'll happily API any needed parts for someone to make an addon to further configure the siege feature.

**Comment by RoboMWM** [comment]
Just an fyi: version 20 will not include the siege feature. Someone will be able to make an addon to provide this functionality though. This ticket will remain open until someone creates such an addon.
